WordPress/src/main/java/org/wordpress/android/userflags/JetpackLocalUserFlagsFlag.kt

```diff
@@ -8,5 +8,5 @@ class JetpackLocalUserFlagsFlag @Inject constructor(
     private val jetpackLocalUserFlagsFeatureConfig: JetpackLocalUserFlagsFeatureConfig,
     private val buildConfigWrapper: BuildConfigWrapper
 ) {
-    fun isEnabled() = jetpackLocalUserFlagsFeatureConfig.isEnabled() && buildConfigWrapper.isJetpackApp
+    fun isEnabled() = buildConfigWrapper.isJetpackApp
```

Comment:
Same question about the removed feature config check.

Reply:
Same as: https://github.com/wordpress-mobile/WordPress-Android/pull/17398#discussion_r1011027792